scripts/image_build.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+export GOPATH=/go
+export PATH=/miniconda/bin:/miniconda/envs/sqlflow-dev/bin:/usr/local/go/bin:/go/bin:$PATH
+
+# 0. Install miniconda and python and python dependencies.
+curl -sL https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -o mconda-install.sh
+bash -x mconda-install.sh -b -p miniconda
+rm mconda-install.sh
+/miniconda/bin/conda create -y -q -n sqlflow-dev python=3.6 ${CONDA_ADD_PACKAGES}
+echo ". /miniconda/etc/profile.d/conda.sh" >> ~/.bashrc
+echo "source activate sqlflow-dev" >> ~/.bashrc
+# keras.datasets.imdb only works with numpy==1.16.1
```

Comment:
Move to `numpy==1.16.1 \ # keras.datasets.imdb only works with numpy==1.16.1`

Reply:
Tested, this will cause `ERROR: Invalid requirement: '#'`

Reply:
I see. Please ignore my comment.